packages/account-v2/tailwind.config.ts

```diff
@@ -1,8 +1,24 @@
 import type { Config } from 'tailwindcss';
 import QuillTailwindConfig from '@deriv/quill-design/quill-tailwind/tailwind.config.cjs';
+const plugin = require('tailwindcss/plugin');
 
 export default {
     content: ['./src/**/*.{js,jsx,ts,tsx}'],
+    plugins: [
+        plugin(({ addUtilities }) => {
+            addUtilities({
+                '.outline': {
+                    'outline-style': 'solid',
+                },
+                '.outline-1': {
+                    'outline-width': '1px',
+                },
+                '.pl-800': {
+                    'padding-left': '16px',
+                },
+            });
+        }),
+    ],
```

Comment:
is it okay to add these in account-v2 alone as per our requirements? or would it be better to inform quill squad to add these things as per figma design?

Reply:
@shahzaib-deriv what is the goal to add this here if we have these styles in tailwind already?
https://tailwindcss.com/docs/outline-style
https://tailwindcss.com/docs/outline-width
pl-`800` is `padding-left: 16px` in `quill-design`

Reply:
these styles don't works for me after struggling for hours the only thing worked was to add them here and I left it here.
I guess the whole issue is overridden classes from tailwind in quill

Reply:
@shahzaib-deriv With @george-usynin-deriv just checked in `account-v2` class `pl-800` from `quill-design` and it works fine (check the image attached). I don't know what's exactly wrong, but I'm not sure somebody could create class with name pl-800 in deriv-app and overwrite this style. Anyway, I'm not sure it's a good approach just adding the same classes as utility classes. Please, recheck your config, because all is working fine from our side.

![image](https://github.com/binary-com/deriv-app/assets/120570511/a24a97dc-23dc-4fc3-93bc-b0a737254f8a)